tests/console/kdump_and_crash.pm

```diff
@@ -27,15 +27,13 @@ sub run() {
     # restart to activate kdump
     script_run 'reboot', 0;
     $self->wait_boot;
-    reset_consoles;
     select_console 'root-console';
 
     # often kdump could not be enabled: bsc#1022064
     return 1 unless kdump_is_active;
     do_kdump;
     # wait for system's reboot
     $self->wait_boot;
-    reset_consoles;
```

Comment:
@okurz  I think in this test are "not needed" and used here with older implementations of wait_boot. Btw wait_boot reset consoles itself.